docs/values.rst

```diff
@@ -120,6 +120,21 @@ instead of using the name of the :class:`~Value` instance.::
     class Dev(Configuration):
         TIME_ZONE = values.Value('UTC', environ_name='MYSITE_TZ')
 
+Allow final value to be used outside the configuration context
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
It was caused by using different hintstyle (hintnone) for bold fonts (which the line of `^`s is for some reason via GH's css - but the line above is not).
Ref: https://github.com/blueyed/dotfiles/blob/7c4d7be86beea22dc7e230762de93790617e16f6/config/fontconfig/fonts.conf#L151

This should not really cause the width to become different, which might be a bug in fontconfig or the webengine backend in Qutebrowser.